jira/client.py

```diff
@@ -92,7 +92,8 @@ def emit(self, record):
 #    "which means that there is a big change of having problems. " \
 #    "Possible workaround http://stackoverflow.com/a/17628350/99834" % encoding)
 
-logging.getLogger('jira').addHandler(NullHandler())
+logger = logging.getLogger('jira')
```

Comment:
I think that the pattern is to use `logging.getLogger(__name__)`  like in  https://github.com/kennethreitz/requests/blob/5aa66615870beeb7b0ed3af27457f689db262ab3/requests/packages/urllib3/contrib/appengine.py#L68